src/modules/itemCounter.js

```diff
@@ -0,0 +1,10 @@
+import { getGame } from './api.js';
+
+const gameCounter = document.getElementById('games');
+
+export default async function gamecount() {
+  const data = await getGame();
+  const count = data.length;
+  console.log(count);
```

Comment:
[OPTIONAL]

- [ ] Kindly remove any console messages before merging this into dev, it shouldn't include any console logs.